projects/Amlogic/bootloader/mkbootimg

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+DTB_BLOBS=($(ls arch/$TARGET_KERNEL_ARCH/boot/dts/amlogic/*.dtb 2>/dev/null || true))
+DTB_BLOBS_COUNT="${#DTB_BLOBS[@]}"
+DTB_BLOB_OUTPUT="arch/$TARGET_KERNEL_ARCH/boot/dtb.img"
+ANDROID_BOOTIMG_SECOND="--second $DTB_BLOB_OUTPUT"
+
+if [ "$DTB_BLOBS_COUNT" -gt 1 ]; then
+  $TOOLCHAIN/bin/dtbTool -o arch/$TARGET_KERNEL_ARCH/boot/dtb.img -p scripts/dtc/ arch/$TARGET_KERNEL_ARCH/boot/dts/amlogic/
```

Comment:
`-o $DTB_BLOB_OUTPUT `?

Why not create `dtb.img` even if it contains a single dtb (so that you don't have a different path for `$DTB_BLOBS_COUNT -eq 1`), does that not work?

Speaking of DTB_BLOBS_COUNT, why are you treating ` $DTB_BLOBS_COUNT ` as a string (surrounded by double quotes) when performing a numeric comparison? I'm pretty sure toy can lose the double quotes in line 5 too.

Reply:
@MilhouseVH I'm simply moving code, I have no intention of updating it for this dead platform

Reply:
Fair enough.